osu.Game.Tests/Visual/Online/TestSceneChatOverlay.cs

```diff
@@ -520,7 +534,7 @@ public void TestHighlightWhileChatHidden()
                 });
             });
 
-            AddStep("Highlight message and show chat", () =>
+            AddStep("Highlight message and open chat", () =>
             {
                 chatOverlay.HighlightMessage(message, channel1);
                 chatOverlay.Show();
```

Comment:
Unrelated but I find it weird that calling `HighlightMessage` doesn't automatically show the chat overlay...

Reply:
I was initially going to make `HighlightMessage` show the chat overlay and call it `ShowHighlighted`, but then reconsidered that when I added the test cases since the chat overlay would already be shown in that case. Either way works I think.